docker/zcash-params/Dockerfile

```diff
@@ -23,12 +23,12 @@ RUN apt-get -qq update && \
 
 # Optimize builds. In particular, regenerate-stateful-test-disks.yml was reaching the
 # GitHub Actions time limit (6 hours), so we needed to make it faster.
-ENV RUSTFLAGS -O
+ENV RUSTFLAGS "-C opt-level=0"
```

Comment:
This might speed up or slow down build time, depending on the amount of extra data produced by un-optimised builds. But we need the optimisations to speed up the runtime parameter download.

I think this will override the `-C opt-level=2` implied by `cargo run --release`, but I'm not sure.